Dockerfile

```diff
@@ -55,8 +55,7 @@ ENV GOVUK_APP_DOMAIN="http://localhost:3000" \
 WORKDIR /app
 COPY . .
 
-RUN yarn jest --passWithNoTests && \
```

Comment:
I'm not sure why this `jest` line is here?

Reply:
I think in ancient days we were not sure how we would test front-end, and jest might have come from whichever place we copy-pasted the dockerfile from :P